js_modules/dagster-ui/packages/ui-core/src/partitions/OrdinalPartitionSelector.tsx

```diff
@@ -0,0 +1,180 @@
+import {
+  Box,
+  Checkbox,
+  Colors,
+  Icon,
+  Menu,
+  MenuDivider,
+  MenuItem,
+  MiddleTruncate,
+  TagSelectorDropdownItemProps,
+  TagSelectorDropdownProps,
+  TagSelectorWithSearch,
+} from '@dagster-io/ui-components';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import {PartitionStatusHealthSource} from './PartitionStatus';
+import {AssetPartitionStatusDot} from '../assets/AssetPartitionList';
+import {partitionStatusAtIndex} from '../assets/usePartitionHealthData';
+import {RunStatus} from '../graphql/types';
+import {RunStatusDot} from '../runs/RunStatusDots';
+
+export const OrdinalPartitionSelector = ({
```

Comment:
Would be nice to add some tests for this component

Reply:
Haha yeah looks like you didn't write any when we added this to `DimensionRangeWizard` ;-) I will add some!

Reply:
ah yeah our testing game was weak then. I think tests for `TagSelector` would be neat. I've been slowly adding more tests to the surfaces I touch (eg. recently for the RunTimeline) so would be good to slowly do that over the time when we touch older components